Gems/ScriptCanvas/Code/Tools/TranslationGeneration.cpp

```diff
@@ -837,7 +849,7 @@ namespace ScriptCanvasEditorTools
                 Helpers::GetTypeNameAndDescription(parameter->m_typeId, argumentName, argumentDescription);
 
                 argument.m_typeId = argumentKey;
-                argument.m_details.m_name = parameter->m_name;
+                argument.m_details.m_name = behaviorMethod->GetArgumentName(argIndex)  ? *behaviorMethod->GetArgumentName(argIndex) : argumentName;
```

Comment:
Instead of invoking the GetArgumentName function twice, the reuslt could be stored in a `const char*` and then just checked twice.